docs/installation/index.md

```diff
@@ -4,7 +4,7 @@ position: 0
 description: How to install the central Octopus Deploy server.
 ---
 
-This section walks you through installing the self-hosted central Octopus Deploy server.
+This section walks you through installing the self-hosted central Octopus Deploy server. If you're interested in the cloud-hosted Octopus server, you can [register for an account](https://octopus.com/account/register).
```

Comment:
Register for an account seems a bit passive. Should we say "sign up" or "launch your own instance now" or something?

Reply:
Launch your own instance now, sounds good.

Reply:
I went with: launch your own cloud instance now.
I added in cloud because that's the installation page. I also moved the sentence into a hint-alert and added emphasis to self-hosted and cloud-hosted, so that readers don't miss them and get confused.